src/models/transactions/escrowCreate.ts

```diff
@@ -0,0 +1,54 @@
+import { ValidationError } from "../../common/errors";
+import { BaseTransaction, GlobalFlags, verifyBaseTransaction } from "./common";
+
+export interface EscrowCreateFlags extends GlobalFlags {
+    tfPassive?: boolean;
+    tfImmediateOrCancel?: boolean;
+    tfFillOrKill?: boolean;
+    tfSell?: boolean;
+}
```

Comment:
Are we doing booleans instead of ints?

Reply:
Yes, we're doing booleans so that you can describe Flags as
```js
{
tfPassive: true,
tfFillOrKill: true,
}
```
It makes it easier to specify flags. We'll also have to specify a constant w/ ints, but that will only be used for processing.